src/app/component/auth/components/sign-up/sign-up.component.ts

```diff
@@ -124,51 +126,52 @@ export class SignUpComponent implements OnInit {
   }
 
   private signInWithGoogleError(errors: HttpErrorResponse): void {
-    try {
-      errors.error.forEach(error => {
-        if (error.name === 'email') {
-          this.emailErrorMessageBackEnd = error.message;
-        } else if (error.name === 'password') {
-          this.passwordErrorMessageBackEnd = error.message;
-        }
-      });
-    } catch (e) {
+    if (!errors.error[0]) {
```

Comment:
change it - your code will fail on an empty array

Reply:
> change it - your code will fail on an empty array

Fixed